blocks/related-resources/related-resources.js

```diff
@@ -33,15 +34,19 @@ async function setRowDetails(row, block) {
           break;
         case 'H4':
         case 'H5':
-          row.assetType = el.innerHTML;
+          row.assettype = el.innerHTML;
           break;
         case 'P':
-          row.description = `<p>${el.innerHTML}</p>${row.description}`;
+          description = `<p>${el.innerHTML}</p> ${description}`;
```

Comment:
There's an extra paragraph tag showing up on the 2nd card of each row I think.
![image](https://user-images.githubusercontent.com/3883795/233726830-00ff8bb7-0675-421e-8e7f-5f4557b15f5c.png)

Reply:
Update: actually this is not a new regression, it's there on the Main branch as well. https://main--merative2--hlxsites.hlx.page/drafts/chelms/4-ways-case-study

Reply:
I will resolve this convo and you can make a new ticket for that.